ci/docs/build.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# Copyright (c) 2018, NVIDIA CORPORATION.
+#########################################
+# cuML GPU build and test script for CI #
+#########################################
+set -ex
+
+# Logger function for build status output
+function logger() {
+  echo -e "\n>>>> $@\n"
+}
+
+# Set path and build parallel level
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=4
+export CUDA_REL=${CUDA_VERSION%.*}
+export CUDF_VERSION=0.7.*
+export RMM_VERSION=0.7.*
+
+# Set home to the job's workspace
+export HOME=$WORKSPACE
+export DOCS_DIR=/data/docs/html
+
+while getopts "d" option; do
+    case ${option} in
+        d)
+            DOCS_DIR=${OPTARG}
+            ;;
+    esac
+done
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+logger "Check environment..."
+env
+
+logger "Check GPU usage..."
+nvidia-smi
+
+logger "Activate conda env..."
+source activate gdf
+conda install -c rapidsai/label/cuda$CUDA_REL -c rapidsai-nightly/label/cuda$CUDA_REL -c conda-forge \
+    cudf=$CUDF_VERSION rmm=$RMM_VERSION
```

Comment:
Would we need to install `cudatoolkit` due to #533 ?

Reply:
I believe that is only for conda builds? It wasn't added to our gpu script. @raydouglass can you confirm if this is needed or not?